modules/test/playwright/tests/client-extension-web/js.spec.ts

```diff
@@ -29,6 +31,11 @@ const SAMPLES = [
 		name: 'Liferay Sample Global JS 2',
 		url: '/o/liferay-sample-global-js-2/global.a0ff2e4a08889609cd1e.js',
 	},
+	{
+		erc: 'LXC:liferay-sample-global-js-3',
+		name: 'Liferay Sample Global JS 3',
+		url: '/o/liferay-sample-global-js-3/global.8c51c23d2fbc94a8abfa.js',
```

Comment:
Hey @thiago-buarqque , won't the hash in this file name change everytime we deploy the CX?

Reply:
@ethib137 No, that hash is built based on the file content. If you scroll up a little bit you'll see we have a fixed hash for the samples 1 and 2 as well